lib/sensors/src/UltraSonicSensor.cpp

```diff
@@ -0,0 +1,33 @@
+#include "UltraSonicSensor.h"
+
+UltraSonicSencor::UltraSonicSensor(PinName echoPin, PinName trigPin) : _echoPin(echoPin), _trigPin(trigPin) {}
+
+uint64_t UltraSonicSensor::pulseIn() {
```

Comment:
bad function name. maybe something like `readPulseWidth` or depending on what the intention here is, `readAveragePulseWidth`.